class/defaults.yml

```diff
@@ -3,6 +3,13 @@ parameters:
     namespace: openshift-logging
     channel: 'stable'
     alerts: 'master'
+    predict_elasticsearch_storage_alert:
+      enabled: true
+      lookback_range: 72h
+      predict_hours_from_now: 72
+      threshold: 85
+      for: 6h
+      severity: warning
```

Comment:
I'm somehow not a fan of the naming. The `alerts` and `ignore_alerts` already feel "randomly thrown in". Maybe we could put the `predict_elasticsearch_storage_alert` in a `custom_alerts` directory, to give the config some kind of structure?

Reply:
I see the issue and feel the same. The problem with putting it under `custom_alerts` is, I'd naively expect to be able to customise everything then from the yaml. Do we want that?

Reply:
Hmm I see your point and I don't think we want to enable to customize everything, that seems too finicky.

So I don't have a better naming suggestion. Let's leave it as is.